frontend/src/components/opportunity/OpportunityCTA.tsx

```diff
@@ -0,0 +1,58 @@
+import { OpportunityStatus } from "src/types/opportunity/opportunityResponseTypes";
+
+import { useTranslations } from "next-intl";
+import { ReactNode } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+export const OpportunityContentBox = ({
+  title,
+  content,
+}: {
+  title?: string | ReactNode;
+  content: string | ReactNode;
+}) => {
+  return (
+    <div className="border radius-md border-base-lighter padding-x-2">
+      {title && <p className="text-bold margin-bottom-0">{title}</p>}
+      <p className="desktop-lg:font-sans-sm margin-top-0">{content}</p>
+    </div>
+  );
+};
+
+const OpportunityCTA = ({
+  status,
+  id,
+}: {
+  status: OpportunityStatus;
+  id: number;
+}) => {
+  const t = useTranslations("OpportunityListing.cta");
+
+  const contentKey = status === "posted" ? "apply_content" : "closed_content";
```

Comment:
I don't think you want to only display this for posted opportunities. There are opportunities that you can apply for that are not in the `posted` state.

Reply:
yeah, as discussed in the comments, I had the requirements wrong, and there isn't any dynamic behavior here, so all the states will show text relating to application